airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/manifest_only_connectors.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+"""This module groups steps made to run tests for a specific manifest only connector given a test context."""
+
+from pipelines.airbyte_ci.connectors.build_image.steps.manifest_only_connectors import BuildConnectorImages
+from pipelines.airbyte_ci.connectors.consts import CONNECTOR_TEST_STEP_ID
+from pipelines.airbyte_ci.connectors.test.context import ConnectorTestContext
+from pipelines.airbyte_ci.connectors.test.steps.common import AcceptanceTests, IncrementalAcceptanceTests, LiveTests
+from pipelines.consts import LOCAL_BUILD_PLATFORM
+from pipelines.helpers.execution.run_steps import STEP_TREE, StepToRun
+
+
+def get_test_steps(context: ConnectorTestContext) -> STEP_TREE:
```

Comment:
💅 I'm wondering if we could find a way to have a common tree for low_code/manifest/python connectors and just add unit tests / integration tests for low_code and python connectors.

Reply:
yes, but these changes are in the rather distant future, aren't they? I imagine we'll be reworking the step tree long before then. Anyway, it's not blocking for me, so I'll let you do what you think is wise :)

Reply:
Theyre not too far away!

Also the way were likely to do "upload for pyairbyte" changes.

So the types of tests we run are even different

https://airbytehq-team.slack.com/archives/C02TL38U5L7/p1719448571131109

Thanks for the push here btw.